app/scss/_side-drawer.scss

```diff
@@ -0,0 +1,88 @@
+.sidedrawer-left, .sidedrawer-center{
+	background-color: $charcoal;
+}
+
+.sidedrawer-header{
+	height: 148;
+	background-color: $accent;
+	width: 100%;
+}
+.sidedrawer-left .sidedrawer-header{
+	padding:16 16 0 16;;
+}
+.sidedrawer-center .sidedrawer-header{
+	padding:20 15 0 15;;
+}
+
+.sidedrawer-header-image{
+	background-color: $grey;
+}
+.sidedrawer-left .sidedrawer-header-image{
+	height: 64;
+	width: 64;
+	border-radius: 32;
+	horizontal-align:left;
+	margin-bottom:36;
+}
+.sidedrawer-center .sidedrawer-header-image{
+	height: 74;
+	width: 74;
+	border-radius: 37;
+	horizontal-align:center;
+	margin-bottom:24;
+}
+
+.sidedrawer-header-brand{
+	color: $grey;
+}
+.sidedrawer-left  .sidedrawer-header-brand{
+	horizontal-align:left;
+	font-size: 14;
+}
+.sidedrawer-center .sidedrawer-header-brand{
+	horizontal-align:center;
+	font-size: 15;
+}
+
+.sidedrawer-list-item{
+	height: 48;
+	horizontal-align:left;
+	width: 100%;
+	orientation:horizontal;
+	color:$btn-color-disabled;
+
+	&.active{
+		color:$accent;
+		background-color: #2b2b2b;
+	}
+}
+
+.sidedrawer-list-item-icon{
+	width:24;
+	horizontal-align:left;
+	font-size: 20;
+	height: 48;
```

Comment:
`height` here might have conflict with inheritance of styles, Had to remove in my app but no time right now to dig in to uncover the exact issues (just a reminder if it comes up in future about the icon not being aligned with the text)